force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,7 +50,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        Integer result;
+        result = a - b;
```

Comment:
You don't have to separate declaring the variable and assigning the value. You can instantiate it with the value like this: `Integer result = a - b;`